README.md

```diff
@@ -2,143 +2,171 @@
+   have broad support are much more likely to make progress than those that
+   don’t receive any comments.
+
+3. If the proposal receives non-trivial interest from community members and
+   generally positive feedback, you can prepare a Pull Request:
+   - Fork this repo.
+   - Create your proposal as `proposals/my-feature.md`
+     (where "my-feature" is descriptive).
+   - Submit a pull request. Make sure to link to the discussion thread.
+
+4. Eventually, the [core team] will decide whether the RFC is a candidate
+   for inclusion in Astro.
+   - An RFC can be modified based upon feedback from the [core team] and
+     community. Significant modifications may trigger a new final comment
+     period.
+   - An RFC may be rejected after public discussion has settled and comments
+     have been made summarizing the rationale for rejection. A member of the
+     [core team] should then close the RFC’s associated pull request.
+   - An RFC may be accepted at the close of its final comment period. A
+     [core team] member will merge the RFC’s associated pull request, at which
+     point the RFC will become 'active', existing in the [proposals] directory.
+   - Once the RFC is implemented, it will be updated with the appropriate
+     **Implementation PR**.
 
 ## Details on Active RFCs
 
-Once an RFC becomes active then authors may implement it and submit the
-feature as a pull request to the Astro repo. Becoming 'active' is not a rubber
-stamp, and in particular still does not mean the feature will ultimately
-be merged; it does mean that the core team has agreed to it in principle
-and are amenable to merging it.
+Once an RFC becomes active then authors may implement it and submit the feature
+as a pull request to the Astro repo. Becoming 'active' is not a rubber stamp,
+and in particular still does not mean the feature will ultimately be merged; it
+does mean that the core team has agreed to it in principle and are amenable to
+merging it.
 
-Furthermore, the fact that a given RFC has been accepted and is
-'active' implies nothing about what priority is assigned to its
-implementation, nor whether anybody is currently working on it.
+Furthermore, the fact that a given RFC has been accepted and is 'active' implies
+nothing about what priority is assigned to its implementation, nor whether
+anybody is currently working on it.
 
-Modifications to active RFC's can be done in followup PR's. We strive
-to write each RFC in a manner that it will reflect the final design of
-the feature; but the nature of the process means that we cannot expect
-every merged RFC to actually reflect what the end result will be at
-the time of the next major release; therefore we try to keep each RFC
-document somewhat in sync with the language feature as planned,
-tracking such changes via followup pull requests to the document.
+Modifications to active RFC’s can be done in followup PR’s. We strive to write
```

Comment:
This quotation mark is very clearly driving me insane!!!

Reply:
That’s an apostrophe. Many computers make this mark automatically. Most browsers search for all of them when you search for any of them.

Reply:
Incredible!